Tests/CheckoutNetworkTests/CheckoutNetworkClientTests.swift

```diff
@@ -88,7 +88,7 @@ final class CheckoutNetworkClientTests: XCTestCase {
         let testResponseCode = 300
         
         let expectedData = "nothing".data(using: .utf8)
-        let expectedResponse = HTTPURLResponse(url: FakePath.testServices.url(),
+        let expectedResponse = HTTPURLResponse(url: FakePath.testServices.url!,
```

Comment:
```suggestion
        let url = try XCTUnwrap(FakePath.testServices.url)
        let expectedResponse = HTTPURLResponse(url: url,
```
Same for other test cases that has !

Reply:
discussed offline